extensions/amp-subscriptions-google/0.1/amp-subscriptions-google.js

```diff
@@ -111,11 +111,11 @@ export class GoogleSubscriptionsPlatform {
       this.handleAnalyticsEvent_.bind(this)
     );
 
-    // Map AMP experiment to SwG experiment.
-    const swgExperiments = {};
-    if (isExperimentOn(ampdoc.win, 'gpay-api')) {
-      swgExperiments.experiments = ['gpay-api'];
-    }
+    // Map AMP experiments to SwG experiments.
```

Comment:
Do we need to worry about namespace collisions?  I don't think it's a problem today but it might be a good idea to preemptively deal it it so somebody doesn't accidentally create an AMP experiment that maps to a swg on they didn't know existed.  

I suggest maybe  find AMP experiments that prefix swg- and then pass them, stripped of the prefix to swg (so experiment swg-foobaz in amp becomes experiment foobaz in swg)

Reply:
Yeah, I was worried about namespace collision. Took your suggestion and prefixed with 'swg-'.